src/main/java/team/baymax/logic/parser/ArgumentMultimap.java

```diff
@@ -25,8 +29,11 @@
      * @param prefix   Prefix key with which the specified argument value is to be associated
      * @param argValue Argument value to be associated with the specified prefix key
      */
-    public void put(Prefix prefix, String argValue) {
+    public void put(Prefix prefix, String argValue) throws CommandException {
```

Comment:
I think this should be ParserException! Since the code is located in the parser package. CommandException is thrown when error occurs in the command classes.